tests/ja4-rules-disabled/test.yaml

```diff
@@ -0,0 +1,9 @@
+requires:
+  min-version: 8.0.0
+  files:
+    - rust/src/ja4.rs
+
+args:
+  - -k none
+
+exit-code: 1
```

Comment:
A comment explaining the purpose of the test is welcome (or a README.md)

Reply:
Done.